objects/obj_en_fleet/Alarm_1.gml

```diff
@@ -508,7 +508,7 @@ if (obj_controller.faction_status[2]="War") and (action="") and (trade_goods="")
                 pnear=instance_nearest(fnear.x,fnear.y,obj_star);
                 
                 if (instance_exists(pnear)) and (instance_exists(orbiting)){
-                    if (fdis<=500) and (pnear.orbiting!=orbiting){// Case 1; really close, wait for them to make the move
+					if (fdis<=500) and (pnear!=orbiting){// Case 1; really close, wait for them to make the move
```

Comment:
can fdis and pnear have better names?

Reply:
Done. Renamed other 3 vars on the file while I was on it.